provisioning/miniprov/hare_mp/utils.py

```diff
@@ -98,28 +98,44 @@ def _save_drive_info(self, path: str):
         disk_key = path.strip('/')
         self.kv.kv_put(f'{hostname}/{disk_key}', drive_info)
 
+    def is_motr_component(self, machine_id: str) -> bool:
+        """
+        Returns True if motr component is present in the components list
+        for the given node>{machine_id} according to the
+        ConfStore (ValueProvider).
+        """
+        comp_names = self.provider.get(f'node>{machine_id}>'
+                                       f'components')
+        for component in comp_names:
+            if(component.get('name') == Const.COMPONENT_MOTR.value):
+                rc = True
+                break
+            else:
+                rc = False
```

Comment:
It doesn't make sense to assign rc to False at every iteration.

Reply:
There can be sequence like this through which for loop will iterate through :
Example 1 : utils, motr, hare
Example 2 : utils, s3, hare
and change of sequences of component names in each of above examples.
This function is so written to work well in any such condition.